doc/nrf/libraries/bluetooth/services/wifi_prov_ble.rst

```diff
@@ -0,0 +1,117 @@
+.. _wifi_prov_readme:
+
+Wi-Fi Provisioning BLE Transport
+################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This library implements the Bluetooth® GATT transport layer for the Wi-Fi® provisioning service.
+It provides the BLE-specific implementation of the transport interface defined by the core Wi-Fi provisioning library.
+The BLE transport layer is designed to work with the core Wi-Fi provisioning library located at ``subsys/net/lib/wifi_prov/``.
+
+Overview
+********
+
+The BLE transport layer is responsible for:
+
+* GATT service interface: Defines the GATT service and serves as the transport layer for the provisioning protocol.
+* BLE-specific message handling: Implements the transport functions required by the core library.
+* Connection management: Manages BLE connections and handles characteristic notifications/indications.
+
+The transport layer implements the transport interface defined by the core Wi-Fi provisioning library:
+
+* ``wifi_prov_send_rsp()``: Sends Response messages via BLE indications
+* ``wifi_prov_send_result()``: Sends Result messages via BLE notifications
```

Comment:
Are these functions that could be linked? If yes, then it could be linked like :c:func:wifi_prov_send_rsp().
```suggestion
* ``wifi_prov_send_rsp()``: Sends Response messages through BLE indications.
* ``wifi_prov_send_result()``: Sends Result messages through BLE notifications.
```